play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -132,8 +132,18 @@ basic computer inquiry or operation. +10% to ONE pertinent class skill
 (hacking, etc.) while using this feat, to be chosen at the time of taking 
 this feat.
 
-First Shot Counts: 
+Fish To Water
 Prerequisites:

```

Comment:
can we add a prereq like... FORT 3? It probably makes more sense to have dex, but i don't think dex needs to be any more valuable

Reply:
I want to add a prereq so that we can favor less powerful primary stats and/or make a min-maxing player's decisions more impactful

Reply:
Yeah, maybe a 3 in Fort.